conda_smithy/templates/build_steps.sh.tmpl

```diff
@@ -62,6 +62,10 @@ setup_conda_rc "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
 {% if build_setup -%}
 {{ build_setup }}{% endif -%}
 {% if yum_build_setup is defined -%}
+# Due to https://bugzilla.redhat.com/show_bug.cgi?id=1537564 old versions of rpm
+# are drastically slowed down when the number of file descriptors is very high.
+# This can be visible during a `yum install` step of a feedstock build.
+ulimit -n 1024
 {{ yum_build_setup }}{% endif -%}
```

Comment:
(unresolved for visibility)

Reply:
> Alternatively, we could also put this is in yum_build_setup directly

Handling in PR: https://github.com/conda-forge/conda-smithy/pull/1961

This also fixed a formatting issue Jinja blank space cleanup otherwise introduces